submissions/kunmi02/week1solution/fizzbuzz.java

```diff
@@ -0,0 +1,22 @@
+public class Fizzbuzz {    
+    public static void main(String[] args) {
+        // TODO code application logic here
+        int endcount=100; //end of my counter
+        for (int i=1;i<=endcount;i++){ //initialized a for loop to count from 1 - 100
+            
+            if ((i % 3)==0 && (i % 5)==0){ // condition for fizzbuzz
+                System.out.println ("FizzBuzz" );
```

Comment:
This is a method call, it be `out.println();` not `out.println ();` Can you verify this error

Reply:
@codedentwickler Boss!! have rectified the formatting issue. please go through sir!!